.github/workflows/build-and-test.yaml

```diff
@@ -36,7 +36,14 @@ jobs:
           target-packages: ${{ steps.get-self-packages.outputs.self-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Check file existence
+        id: check_files
+        uses: andstor/file-existence-action@v1
+        with:
+          files: lcov/total_coverage.info,coveragepy/.coverage
```

Comment:
Is this `or` condition or `and` condition?

Reply:
Regarding to the document, this is `and` condition. I will consider to change here to `or` condition.
https://github.com/andstor/file-existence-action#example

Reply:
I think you can write a simple bash script instead. :+1:

Reply:
I changed to use bash script to check file existence: c879100